### PR TITLE
ADD userDao interface + impl

### PR DESCRIPTION
**Pull request by tomasedl**

### CODE REVIEW

File: src/main/java/cz/muni/pa165/teamwhite/formula1/dao/UserDao.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import java.util.List;
+
+
+/**
+ * @author Tomas Sedlacek
+ */
+public interface UserDao {
+
+    /**
+     * This method create new user in database
```
Comment (khecova) [inline src/main/java/cz/muni/pa165/teamwhite/formula1/dao/UserDao.java:13]: melo by byt creates

File: src/main/java/cz/muni/pa165/teamwhite/formula1/dao/UserDao.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+     * This method finds user in DB by entered id
+     * @param id - id of the requested user
+     * @return found user
+     */
+    User findById(Long id);
+
+    /**
+     * This method finds user in DB by entered login
+     * @param login - login of the requested user
+     * @return found yser
```
Comment (khecova) [inline src/main/java/cz/muni/pa165/teamwhite/formula1/dao/UserDao.java:28]: typo - "yser"

**Review by khecova — COMMENTED** [review]
nejake preklepy jinak asi ok :)

**Review by Jakub-Fajkus — APPROVED** [review]

**Review by khecova — APPROVED** [review]
ok

**Pull request merged**